assets/src/styles/blocks/TakeActionBoxout/edit.scss

```diff
@@ -0,0 +1,12 @@
+.boxout {
+  position: relative;
+
+  a {
+    pointer-events: none;
+  }
+
+  // Prevent line clamp on editable stuff, because it's very buggy.
+  [contenteditable] {
+    -webkit-line-clamp: unset !important;
```

Comment:
`!important` seems justified here, because currently there is no way that it can work properly on `contenteditable` elements.